invenio_communities/views/api.py

```diff
@@ -177,9 +181,21 @@ def get(self, community_id):
         community = Community.get(community_id)
         if not community:
             abort(404)
-        return self.make_response(
-                community, links_item_factory=default_links_item_factory)
 
+        etag = hashlib.sha1(
+                '{0}__{1}'.format(community.id, community.updated).encode(
+                        'utf-8')).hexdigest()
+        try:
+            self.check_etag(etag)
+            response = self.make_response(
+                    community, links_item_factory=default_links_item_factory)
+        except SameContentException:
```

Comment:
Do we need this exception handler here? `SameContentException` should inherit from `HTTPException` and set the status code and headers automagically.